### PR TITLE
Rename datastore to migrations in biome

### PR DESCRIPTION
**Pull request by RyanLassigBanks**

### CODE REVIEW

File: libsplinter/src/biome/migrations/mod.rs [hunk 1]
```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Provides sql migration scripts and methods for executing
```
Comment (ltseeley) [inline libsplinter/src/biome/migrations/mod.rs]: This commit says "for biome", but it should say "for biome migrations"

Comment (RyanLassigBanks) [reply libsplinter/src/biome/migrations/mod.rs]: Updated commit message

File: cli/src/action/database.rs [hunk 2]
```diff
@@ -18,7 +18,7 @@ use super::Action;
 use crate::error::CliError;
 use diesel::{connection::Connection as _, pg::PgConnection};
 #[cfg(feature = "database-migrate-biome")]
-use splinter::biome::datastore::run_postgres_migrations;
```
Comment (agunde406) [inline cli/src/action/database.rs]: Typo in commit message "databasestore -> datastore"

Comment (RyanLassigBanks) [reply cli/src/action/database.rs]: Updated

File: libsplinter/src/biome/migrations/mod.rs [hunk 3]
```diff
@@ -12,6 +12,21 @@
...
+//! migrations.
+//!
+//! ```ignore
+//!
+//! use biome::migrations::run_postgres_migratoins;
```
Comment (agunde406) [inline libsplinter/src/biome/migrations/mod.rs]: typo "run_postgres_migratoins;" -> "run_postgres_migrations;"

**Pull request merged**